src/misc.jl

```diff
@@ -201,3 +201,42 @@ end
     root --> :top
     GraphPlot((source, destiny))
 end
+
+
+@userplot AndrewsPlot
+
+"""
+https://en.wikipedia.org/wiki/Andrews_plot
+
+Example
+```julia
+using RDatasets, PlotRecipes
+iris   = dataset("datasets", "iris")
+y      = Array(iris[:,1:4])
```

Comment:
Given that this is in PlotRecipes (that reexports StatPlots) it should be able to plot a DataFrame. Currently, unfortunately, the implementation of DataFrame plotting requires that h.args[1] is tested for being a DataFrame explicitly - I will try to make this easier so it's OK for now.